client/idrepo/console/pom.xml

```diff
@@ -124,7 +124,7 @@ under the License.
     </dependency>
     <dependency>
       <groupId>org.apache.logging.log4j</groupId>
-      <artifactId>log4j-slf4j18-impl</artifactId>
+      <artifactId>log4j-slf4j-impl</artifactId>
```

Comment:
Why this (and similar changes) replacing `log4j-slf4j18-impl` with its "general JDK" counterpart?

Reply:
(Copying the answer here as well so it's available for the public)

> This is to mainly to align with CAS, and to also align with the now-stable slf4j version, to avoid these:

```
java.lang.ClassCastException: class org.slf4j.helpers.NOPLoggerFactory cannot be cast to class 
org.apache.logging.slf4j.Log4jLoggerFactory (org.slf4j.helpers.NOPLoggerFactory and 
org.apache.logging.slf4j.Log4jLoggerFactory are in unnamed module of loader 
org.springframework.boot.loader.LaunchedURLClassLoader
```

Ref: https://github.com/apereo/cas/pull/5261
Ref: https://github.com/apereo/cas/pull/5264

Reply:
Thanks for explanation, LGTM!